content/en/docs/tasks/traffic-management/ingress/service-apis/index.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Kubernetes Service APIs (Experimental)
+description: Describes how to configure the Kubernetes Service APIs with Istio.
+weight: 99
+keywords: [traffic-management,ingress]
+owner: istio/wg-networking-maintainers
+test: no
+---
+
+This task describes how to configure Istio to expose a service outside of the service mesh cluster, using the Kubernetes [Service APIs](https://kubernetes-sigs.github.io/service-apis/). These APIs are an actively developed evolution of the `Service` and `Ingress` APIs.
+
+{{< warning >}}
+Both the APIs, as well as Istio's implementation of the APIs, are currently experimental and intended only for evaluation. They will undergo significant changes in future versions. For most users, we recommend using the [Istio Gateway](/docs/tasks/traffic-management/ingress/ingress-control/).
+{{< /warning >}}
+
+## Setup
+
+First, install the Service APIs CRDs:
+
+{{< text bash >}}
+$ kubectl kustomize "github.com/kubernetes-sigs/service-apis/config/crd?ref=v0.1.0-rc2" | kubectl apply -f -
+{{< /text >}}
+
+Next, install Istio, or reconfigure an existing installation to enable the Service APIs controller:
+
+{{< text bash >}}
+$ istioctl install --set values.pilot.env.PILOT_ENABLED_SERVICE_APIS=true
+{{< /text >}}
+
+Finally, follow the instructions in the [Determining the ingress IP and ports](/docs/tasks/traffic-management/ingress/ingress-control/#determining-the-ingress-ip-and-ports) sections of the [Ingress Gateways task](/docs/tasks/traffic-management/ingress/ingress-control/) in order to retrieve the external IP address of your ingress gateway.
+
+## Configuring a Gateway
+
+See the [Service APIs](https://kubernetes-sigs.github.io/service-apis/) documentation for information about all of the APIs included.
+
+1. Deploy a test application:
+
+    {{< text bash >}}
+    $ kubectl apply -f @samples/httpbin/httpbin.yaml@
+    {{< /text >}}
+
+1. Deploy the Service APIs configuration:
+
+    {{< text bash >}}
+    $ kubectl apply -f - <<EOF
+    apiVersion: networking.x-k8s.io/v1alpha1
+    kind: GatewayClass
+    metadata:
+      name: istio
+    spec:
+      controller: istio.io/gateway-controller
+    ---
+    apiVersion: networking.x-k8s.io/v1alpha1
+    kind: Gateway
+    metadata:
+      name: gateway
+      namespace: istio-system
+    spec:
+      gatewayClassName: istio
+      listeners:
+      - hostname: "*"
+        port: 80
+        protocol: HTTP
+        routes:
+          namespaces:
+            from: All
+          selector:
+            matchLabels:
+              selected: "yes"
+          kind: HTTPRoute
+    ---
+    apiVersion: networking.x-k8s.io/v1alpha1
+    kind: HTTPRoute
+    metadata:
+      name: http
+      namespace: default
+      labels:
+        selected: "yes"
+    spec:
+      hostnames: ["httpbin.example.com"]
+      rules:
+      - matches:
+        - path:
+            type: Prefix
+            value: /get
```

Comment:
Does the `requestHeaderModifier`, below, actually work?
If so, maybe you should change the path to `/headers`, so you can demonstrate it working in the curl example.

Reply:
`/get` shows the header as well. It does work

Reply:
Ah, cool. In that case it would be good to include it in the output (around line 103) and mention it in the following text.